File: HAI704I - Architectures Distribuées/TP2/TP2-Server/TP2-ServiceServer/model/customer/card/CreditCard.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TP2_Server.model.customer.card
{
    [Serializable]
    public class CreditCard
    {
        public string Number
        {
            get;
            set;
        }

        public string ExpirationDate
        {
            get;
            set;
        }

        public string CVV
        {
            get;
            set;
        }
    }
}

File: HAI704I-ArchitecturesDistribuees/TP2/TP2-Java/TP2-Server/src/main/java/tp2/server/model/customer/card/CreditCard.java
package tp2.server.model.customer.card;

public class CreditCard
{
	private String number;
    private String expirationDate;
    private String CVV;
    
    public CreditCard()
	{
    	this("", "", "");
	}
    
	public CreditCard(String number, String expirationDate, String cVV)
	{
		this.number = number;
		this.expirationDate = expirationDate;
		this.CVV = cVV;
	}
	
	public String getNumber()
	{
		return number;
	}
	public void setNumber(String number)
	{
		this.number = number;
	}
	public String getExpirationDate()
	{
		return expirationDate;
	}
	public void setExpirationDate(String expirationDate)
	{
		this.expirationDate = expirationDate;
	}
	public String getCVV()
	{
		return CVV;
	}
	public void setCVV(String cVV)
	{
		CVV = cVV;
	}
    
    
}

